src/configure/test/suite/GitHubClient.test.ts

```diff
@@ -0,0 +1,135 @@
+import { GithubClient } from "../../clients/github/githubClient";
+
+var expect = require('chai').expect;
+var nock = require('nock');
+
+let org = "Organization";
+let repoName = "Repository";
+let githubClient = new GithubClient("PAT", "repoUrl");
+
+
+describe('# Testing listOrganizations() ', function () {
+    this.timeout(500000);
```

Comment:
this is 5s or 500s?

Reply:
500s, but this was put because earlier tests were taking longer to execute but now that issue is resolved so I'll change it back to default timeout value.